istio/Installation.md

```diff
@@ -218,3 +225,10 @@ For product images run
 ```
 oc process -f istio_product_operator_template.yaml | oc delete -f -
 ```
+
+## Removing the istio-operator Project
+Finally, you can remove the `istio-operator` project that you created:
```

Comment:
Not sure you need the headline above. This is part of the process of removing the Istio opreator. 

Might be better to say: 
The  `istio-operator` project can now be removed.